src/amo/components/Addon/index.js

```diff
@@ -388,15 +388,25 @@ export class AddonBase extends React.Component {
       return null;
     }
 
+    {
+      /* Adding wrapping divs here seems to address what we think is a
+      reconcillation issue —— which causes the classname to not always get added
+      correctly (e.g.: when the page is refreshed and the addon has
+      a description).
+      See https://github.com/mozilla/addons-frontend/issues/4744
+    */
+    }
     return (
-      <AddonsByAuthorsCard
-        addonType={addon.type}
-        authorDisplayName={addon.authors[0].name}
-        authorUsernames={addon.authors.map((author) => author.username)}
-        className="Addon-MoreAddonsCard"
-        forAddonSlug={addon.slug}
-        numberOfAddons={6}
-      />
+      <div>
```

Comment:
Does this actually work? You need to make sure you restart your dev server when testing it because that will reload the code running on the server. The client code reloads automatically which would have introduced a new dom element when compared to the server rendered dom and forced a re-render.

Reply:
Yes, I believe I did a full restart because I don't trust the hot reload, I have many weird issues so I often restart.